addons/binding/org.openhab.binding.miele/ESH-INF/thing/xgw3000.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="miele"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Miele Bridge -->
+    <bridge-type id="xgw3000">
+        <label>Miele Bridge</label>
+        <description>The miele bridge represents the Miele@Home XGW3000 gateway.</description>
+
+        <properties>
+        	<property name="vendor">Miele</property>
+        </properties>
+        
+        <config-description>
+            <parameter name="ipAddress" type="text">
+                <context>network_address</context>
+                <label>Network Address</label>
+                <description>Network address of the Miele@Home gateway.</description>
+                <required>true</required>
+            </parameter>
+            <parameter name="interface" type="text">
+                <context>network_address</context>
+                <label>Network Address of the Multicast interface</label>
+                <description>Network address of openHAB host interface where the bindign will listen for multicast events coming from the Miele@Home gateway</description>
+                <required>true</required>
```

Comment:
> Btw, when entering an invalid ip, the binding ends up consuming 100% CPU with millions of messages like

Question: would an invalid IP address be enough of a reason to put the Bridge OFFLINE? 

Currently, i schedule a single Runnable that internally recovers from errors, but that is looping endlessly e.g. while(true). I could put the Bridge OFFLINE and have the Runnable exit for good